### ISSUE TITLE
Nicer web front-end

### ISSUE THREAD

**Issue by leipert** [nareike/adhs#5]
Which features do you want to support in a nicer web front-end?

Maybe a autocompletion editor like [yasgui](http://yasgui.org/)?

and a nice searchable / sortable html table?

**Comment by leipert** [comment]
yasgui actually is made of two parts: 
1. http://yasqe.yasgui.org/ - Editor
2. http://yasr.yasgui.org/ - Result set

**Comment by nareike** [comment]
I have not yet decided what this means. :smiley: (That's why it was on the todo list and not the issue list :stuck_out_tongue: ) But yes, I was thinking about something like yasgui but I have the feeling that this will be a bit too much as a default web front-end.

**Comment by leipert** [comment]
Maybe it's better to stop the web development right here and better create a separate repo which could serve as a nice frontend?

**Comment by nareike** [comment]
Yes, I would rather go into this direction. Actually, yasgui already _is_ a nicer web front end, isn't it? It shouldn't be too hard to set up to work with adhs.

**Comment by seebi** [comment]
I would like to see yasgui as the default frontend instead of the one in sparql.html . no need to invent something new here.

**Comment by leipert** [comment]
that was our idea as well :)
